doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc

```diff
@@ -12,18 +12,25 @@ These steps describe a basic cluster configuration. For more complex configurati
 * {DATAGRID} {DATAGRID_VERSION_LONG} is installed as described in <<clustering-datagrid-proc_{context}>>.
 * AMQ Broker is installed and configured, as described in <<JMS-broker-configure_{context}>>.
 * {EAP} and {PRODUCT} are installed on each node of the cluster as described in <<clustering-download-eap-bc-proc>>.
-* An NFS server with a mounted partition is available.
+* An NFS server with an exported share is available, as described in <<nfs-server-configure_{context}>>.
 
 .Procedure
+. Mount the directory shared over NFS as `/data`. Enter the following commands as the root user:
++
+[subs="attributes,verbatim,macros"]
+----
+mkdir /data
+mount <NFS_SERVER_IP>:/opt/kie/data /data
```

Comment:
what if we change the `/opt/kie/data` with a "variable" like NFS_SERVER_IP so they know they can change the directory?

Reply:
I'm afraid this could make the process too complicated to understand. So I added a note instead,